nixos/modules/services/web-apps/jitsi-meet.nix

```diff
@@ -0,0 +1,497 @@
+        ${config.services.prosody.package}/bin/prosodyctl register jvb auth.${cfg.hostName} "$VIDEOBRIDGE_SECRET"
+
+        cd ${config.services.prosody.dataDir}
+
+        # regenerate the certificates after 1 month - they are valid for a year, that leaves 11 months for service restart
+        find ${cfg.hostName}.crt -mtime +30 -delete || true
+
+        # generate self-signed certificates
+        if [ ! -f ${cfg.hostName}.crt ]; then
+          export PATH=$PATH:${pkgs.openssl.bin}/bin
+          rm -f {,auth.}${cfg.hostName}.{crt,key,cnf}
+          # not sure this one is even used but it avoids some errors in logs
+          echo | ${config.services.prosody.package}/bin/prosodyctl cert generate ${cfg.hostName}
+          # used for jicofo<->prosody user connection
+          echo | ${config.services.prosody.package}/bin/prosodyctl cert generate auth.${cfg.hostName}
+        fi
+
+        # don't know how to add it to the system-wide ceritificate bundle, instead convert it to java truststore and pass it to jicofo
+        rm -f ${dataDir}/prosody-ca.jks
+        ${pkgs.jre}/bin/keytool -importcert -file auth.${cfg.hostName}.crt -keystore ${dataDir}/prosody-ca.jks -storepass javapls -noprompt
+        chown root:jitsi-meet ${dataDir}/prosody-ca.jks
+      '';
+    };
+
+    services.nginx = mkIf cfg.nginx.enable {
+      enable = mkDefault true;
+      virtualHosts.${cfg.hostName} = {
+        root = pkgs.jitsi-meet;
+        locations."~ ^/([a-zA-Z0-9=\\?]+)$" = {
+          extraConfig = ''
+            rewrite ^/(.*)$ / break;
+          '';
+        };
+        locations."/" = {
+          index = "index.html";
+          extraConfig = ''
+            ssi on;
+          '';
+        };
+        locations."/http-bind" = {
+          proxyPass = "http://localhost:5280/http-bind";
+          extraConfig = ''
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header Host $host;
+          '';
+        };
+        locations."=/external_api.js" = {
+          alias = "${pkgs.jitsi-meet}/libs/external_api.min.js";
+        };
+        locations."=/config.js" = {
+          alias = overrideJs "${pkgs.jitsi-meet}/config.js" "config" (recursiveUpdate defaultCfg cfg.config);
+        };
+        locations."=/interface_config.js" = {
+          alias = overrideJs "${pkgs.jitsi-meet}/interface_config.js" "interfaceConfig" cfg.interfaceConfig;
+        };
+      };
+    };
+
+    systemd.services.jitsi-videobridge2 = {
+      aliases = [ "jitsi-videobridge.service" ];
+      description = "Jitsi Videobridge";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      environment.JAVA_SYS_PROPS = attrsToArgs jvbProps;
+
+      serviceConfig = {
+        Type = "exec";
+        ExecStart = "${pkgs.jitsi-videobridge}/bin/jitsi-videobridge ${attrsToArgs jvbArgs}";
+
+        DynamicUser = true;
+        User = "jitsi-videobridge";
+        Group = "jitsi-videobridge";
```

Comment:
If this group isn't used anywhere, you can just drop the line, a temporary one with a less interesting name will be generated - or just make it `jitsi-meet` maybe?

Reply:
Dropping it actually generates the same name:) Replacing it with `jitsi-meet` and dropping `SupplementaryGroups` seems to work fine though.